### DISCUSSION TITLE
Student Developer Pack Email Already Used

### DISCUSSION THREAD

**Discussion by byung806** [community/community#56976] [GitHub Education]
### Hi everyone,

Hello,

My Student Developer Pack expired a few months ago, so I recently filled out applications to renew it. However, they were rejected without a specific reason. I made sure I took a clear photo of my student ID and it clearly had date, name, and school on it. I also made sure my school was part of Github Campus. I also have my full name (matching the ID) in my bio.

I applied using the email I used last time to apply (on the same Github account), but there was a popup saying the email "has already been used to obtain an academic discount on Github account(s) mygithubaccount." I also suspect this is the reason why my application was rejected.

I also looked through the other community posts, but none of them seemed to address the same problem as I have. Do I have to link another email to my account and use the new email to renew my Student Developer Pack or is it possible for me to renew my subscription with this current email?

Thank you!

**Comment by xaviermarquez-alba** [comment]
same problem here.

**Comment by ibrhmkurt** [comment]
same problem

**Reply by samus-aran** [reply]
Hi @ibrhmkurt 👋 

Is that still the case regarding to the application stating the academic email is already in use?

**Comment by dakshithadissanayaka** [comment]
It seems like your application for the Student Developer Pack renewal was rejected due to your email address being previously used to obtain an academic discount. This is in line with GitHub's policy, which allows only one academic discount per email address.

To resolve this issue and renew your Student Developer Pack, you have two options:

1. **Link a Different Email Address:** You can create a new email address and link it to your GitHub account. Once the new email address is verified, you can use it to apply for the Student Developer Pack renewal.

2. **Contact GitHub Support:** If you're unable to create a new email address for any reason, you can contact GitHub support and explain the situation. They may be able to manually process your renewal request using your current email address.

Here's a step-by-step guide on how to link a new email address to your GitHub account:

1. Go to your GitHub account settings: [[https://docs.github.com/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/adding-an-email-address-to-your-github-account](https://github.com/orgs/community/discussions/56976)](https://docs.github.com/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/adding-an-email-address-to-your-github-account)
2. In the left sidebar, click on "Email"
3. Under "Email addresses", click on "Add email address"
4. Enter your new email address and click on "Verify email"
5. Check your new email address for a verification link from GitHub
6. Click on the verification link to confirm your email address

Once your new email address is verified, you can use it to renew your Student Developer Pack subscription.

If you choose to contact GitHub support, follow these steps:

1. Go to the GitHub help center: [https://support.github.com/](https://support.github.com/)
2. Click on "Contact us"
3. Select "I have a problem with my GitHub Education account" as the issue category
4. In the "Description" field, explain that your Student Developer Pack renewal application was rejected due to your email address being previously used. Request that they manually process your renewal request using your current email address.
5. Click on "Submit"

GitHub support will review your request and respond to you as soon as possible.

I hope this information is helpful. Please let me know if you have any other questions.
